app/Makefile

```diff
@@ -15,16 +15,16 @@ help: ## This help message
 .DEFAULT_GOAL := help
 
 build: ## Activate a virtual environment and install application dependencies
-	if [ -d "$(VENV)" ]; then rm -rf $(VENV); fi
-	python3 -m venv $(VENV)
-	$(PIP) install --upgrade pip
-	$(PIP) install wheel
+	@if [ -v ${CLEAN_VENV} ] && [ ${CLEAN_VENV} = "true" ] && [ -d "$(VENV)" ]; then rm -rf $(VENV); fi
```

Comment:
The python virtual environment build in the development environment should never be copied or used within the docker context. So one can adapt the apt/Dockerfile that this environment never will be copied or one removes the environment before one does the copy as it was. The later is of course not the most proper one.